.github/renovate.json

```diff
@@ -1,13 +1,18 @@
 {
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
```

Comment:
Is the problem with automerge just that a reviewer needs to approve?

To reduce noise, we may want to try batched updates instead: https://docs.renovatebot.com/getting-started/use-cases/#batched-updates. There may be some packages that never release a major version change because they don't have breaking changes, but are pushing out new features or fixes frequently.

Reply:
No, the problem is that we have too many PRs without significant updates.

Batched updates should also work well. I've updated as such.